### ISSUE TITLE
Error when closing account (distribute_balance_as_final_gift)

### ISSUE THREAD

**Issue by Changaco** [liberapay/liberapay.com#874]
https://sentry.io/share/issue/1ce6d86ccb7e4067a06d42ad0257b5ce/

```
  File "site-packages/algorithm.py", line 321, in loop
    new_state = function(**deps.as_kwargs)
  File "aspen/request_processor/algorithm.py", line 73, in render_resource
    return {'output': resource.render(state)}
  File "aspen/http/resource.py", line 78, in render
    return super(Dynamic, self).render(available[0], state)
  File "aspen/simplates/simplate.py", line 121, in render
    exec(self.pages[1], context)
  File "/opt/python/bundle/3/app/www/%username/settings/close.spt", line 17, in <module>
    participant.close(disburse_to)
  File "liberapay/models/participant.py", line 465, in close
    self.distribute_balance_as_final_gift(cursor)
  File "liberapay/models/participant.py", line 560, in distribute_balance_as_final_gift
    assert balance == 0  # TODO handle this
AssertionError
```

**Comment by Changaco** [comment]
The user had dollars in his wallet but no USD donations set up.

**Comment by Changaco** [comment]
This was fixed in #1169.
